### ISSUE TITLE
build error with 4.9.0rc2

### ISSUE THREAD

**Issue by wbolster** [DisplayLink/evdi#50]
the `dkms` build of the `evdi` module with kernel 4.9.0rc2 fails with this error message:

```
/var/lib/dkms/evdi/1.2.65/build/evdi_modeset.c: In function ‘evdi_modeset_init’:
/var/lib/dkms/evdi/1.2.65/build/evdi_modeset.c:375:2: error: implicit declaration of function ‘drm_mode_create_dirty_info_property’ [-Werror=implicit-function-declaration]
  drm_mode_create_dirty_info_property(dev);
  ^
cc1: some warnings being treated as errors
scripts/Makefile.build:290: receptet för målet ”/var/lib/dkms/evdi/1.2.65/build/evdi_modeset.o” misslyckades
```

**Comment by displaylink-mlukaszek** [comment]
This should now be fixed with b7606bccc0cb9484c2845a3e556faf60ed3c0bff

**Comment by wbolster** [comment]
that was quick, thanks
